### ISSUE TITLE
It does not use proxy settings in Kiosk mode on Chromebox

### ISSUE THREAD

**Issue by patleg** [matt-cook/kiosk#153]
On a Chromebox, Kiosk cannot display a public web page in an enterprise environment when a proxy is needed. Although specified on the Chromebox settings and working fine on regular mode, the proxy configuration seems not to be used in kiosk mode...
Is it a device problem ? Can it be handled in the code ?
Thanks in advance !

**Comment by dansut** [comment]
It's worse than that, and this isn't a problem with @matt-cook 's kiosk project but with Chrome in general when in Kiosk mode.

I have a whole bunch of Chromeboxes in kiosk mode trapped behind restrictive corporate firewalls and have to jump all sorts of hoops to make them work properly. I've spent hours trying to communicate with Google's 'tech' support for what was then 'Google for Work' to try and get them to take notice of this 'bug' - I believe it is actually a whole group of bugs - only to have them state it is not their problem and redirect me to the Chromium project. I even bought a ChromeOS management license in the hope I might get better support. I actually believe the support teams I got through didn't even have the technical ability to understand the basics of this problem and weren't willing to escalate it to someone who did.

Basically, if any Chrome kiosk mode app **needs** to use a proxy then you are in trouble, the situation is worse if you have ChromeOS devices locked in kiosk mode, there are multiple issues and although working around all of them is possible, there seem to be no current ways to do it in app code - it really needs to be done on the subnet router/server.

I would suggest looking here: Chromium bugs [172285](https://bugs.chromium.org/p/chromium/issues/detail?id=172285), [585657](https://bugs.chromium.org/p/chromium/issues/detail?id=585657)
Also I'd love upvotes to my [stackoverflow question on this issue](http://stackoverflow.com/q/32487520/3448214)

Sorry to be the bearer of bad news :(
